lib/solid.ex

```diff
@@ -143,6 +95,11 @@ defmodule Solid do
   end
 
   defp do_render({:tag, tag}, context, options) do
+    {tag_text, context} = render_tag(tag, context, options)
+    {tag_text, context}
+  end
```

Comment:
Couldn't this be:

```suggestion
  defp do_render({:tag, tag}, context, options) do
    render_tag(tag, context, options)
  end
```

Or is that just done for clarity of what we're returning?

Reply:
I don't remember tbh! Much better this way you suggested thanks